### ISSUE TITLE
Set Username and Password at Run time

### ISSUE THREAD

**Issue by ThBM** [kgierke/nuxt-basic-auth#9]
Hi,

Is it possible to define username and password properties at runtime with env variables?

Let me explain furthermore. Here is what I'm trying to achieve:

```typescript
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
  runtimeConfig: {
    basicAuthUsername: "hardcoded-username", // Will be replace by NUXT_BASIC_AUTH_USERNAME at runtime
    basicAuthPassword: "hardcoded-password", // Will be replace by NUXT_BASIC_AUTH_PASSWORD at runtime
  },

  basicAuth: {
    users: [
      {
        username: useRuntimeConfig().basicAuthUsername,
        password: useRuntimeConfig().basicAuthPassword,
      },
    ],
  },

});
```

However, `useRuntimeConfig` is not defined inside the `nuxt.config.ts` file.
If I switch to `process.env. NUXT_BASIC_AUTH_XXX` it is not replaced at run time and keep the value of build time.

Do you know any solution / workaround to this issue?

Thanks for your help !

Thib

**Comment by kgierke** [comment]
Yes, that's already possible, all configuration parameters can be set trough environment variables, for example to override the users at runtime just use the following environment variable:

`NUXT_BASIC_AUTH_USERS=username:password`.

Essentially all configuration options can be overridden by following the pattern of `NUXT_BASIC_AUTH_<OPTION>`

I will Update the Readme File soon to make it more clear.

**Comment by kgierke** [comment]
Fixed in: [fc0a6f2](https://github.com/kgierke/nuxt-basic-auth/commit/fc0a6f22cd06a72de8d243c05cf8b2530650d3c8)
